generator/typefile.go

```diff
@@ -86,13 +86,23 @@ func (file *typeFile) parseAttributes(namePrefix string, typePrefix string, attr
 		}
 
 		attr := attr[name]
+		// if arrays are allowed, example marker sizes, then javascript allows multiple different inputs hard to map in golang:
+		// [100,50,25,25,50,100] -> use different sizes for the markers
+		// [100] -> only the first marker will have size 100, all others have 0 size
+		// 100 -> all markers have size 100
+		// this can only be captures by using interface and allowing the user to define the used type themselves.
+		// this sadly means that there is no more type checking on those entries
+		var ajust = func(typename string) string { return typename }
+		if attr.ArrayOK {
+			ajust = func(typename string) string { return "interface{}" }
```

Comment:
I understand this, but it would be nice to keep the scope of this PR to create multiple versions. We can merge this later :blush: